clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/SemiAutoClioIndex.scala

```diff
@@ -77,23 +77,15 @@ case object UbamIndex extends BackCompatibleUbamIndex {
   override val commandName: String = "ubam"
 }
 
-// Temporary trait to remain until WgsCram fully phased out
-trait BackCompatibleCramIndex
+case object CramIndex
     extends SemiAutoClioIndex[
       CramKey,
       CramMetadata,
       CramQueryInput
-    ] {
-  override val name: String = "Cram"
-}
-
-case object WgsCramIndex extends BackCompatibleCramIndex with DeliverableIndex {
-  override val urlSegment: String = "wgscram"
-  override val commandName: String = "wgs-cram"
-}
-
-case object CramIndex extends BackCompatibleCramIndex with DeliverableIndex {
+    ]
+    with DeliverableIndex {
   override val urlSegment: String = "cram"
+  override val name: String = "Cram"
   override val commandName: String = "cram"
```

Comment:
Nitpick:
cram Cram cram
seems like we could consolidate these a bit.

Reply:
Yeah, but its like that for every index, so I think we would refactor that in a later ticket. I'm just ready to be done with this API change...